v2_test.go

```diff
@@ -136,6 +136,11 @@ var invalidParseV2Tests = []struct {
 		reader:        newBufioReader(append(append(append(SIGV2, byte(PROXY), byte(TCPv6)), lengthV6Bytes...), fixtureIPv4Address...)),
 		expectedError: ErrInvalidLength,
 	},
+	{
+		desc:          "UNSPEC but no length",
+		reader:        newBufioReader(append(SIGV2, byte(LOCAL), byte(UNSPEC))),
+		expectedError: ErrCantReadLength,
+	},
```

Comment:
Please, add a test with length greater than zero but no TLVs. This will improve code coverage.